pkg/task/restore.go

```diff
@@ -188,7 +188,19 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 
 	// We make bigger errCh so we won't block on multi-part failed.
 	errCh := make(chan error, 32)
-	tableStream := client.GoCreateTables(ctx, mgr.GetDomain(), tables, newTS, errCh)
+	sessionPool := make([]glue.Session, 0, cfg.Concurrency)
```

Comment:
the pool size is 128 by default. good for tidb?

Reply:
Yes, I think so.

> Since the execution time of a DDL has no much relative to environment, maybe a fixed value(like 16 or 32) would be good?